tests/contracts/contracts/migration/PolicyManagerOld.sol

```diff
@@ -719,39 +719,7 @@ contract PolicyManagerOld is Upgradeable {
     /// @dev the `onlyWhileUpgrading` modifier works through a call to the parent `verifyState`
     function verifyState(address _testTarget) public override virtual {
```

Comment:
I am not sure about Solidity syntax, but if it just calls the superclass, does it need to be overridden at all?

Reply:
You're right, it was not necessary, I just wanted to show there's no verifyState check now (and this is hack), but comment will be enough

Reply:
✔️